doc/2/api/protocols/http/index.md

```diff
@@ -64,3 +64,22 @@ The listening port can be modified under the `server.port` section of the [confi
 HTTP and WebSocket protocols share the same underlying server instance.  
 Modifying the listening port will impact these two protocols.
 :::
+
+### HTTP-only options
+
+Only the HTTP protocol does support the following options in a [KuzzleRequest](../../payloads/request/index.md)
+
+| Property         | Type        | Description                                                                 |
+| ---------------- | ----------- | --------------------------------------------------------------------------- |
+| **`cookieOnly`** | **boolean** | **Instruct Kuzzle to store / use authentication token within HTTP Cookies** |
```

Comment:
Our documentation style is as follows:
* property/argument names are to be put between quotes
* types are to be put between `pre` tags 

Not really practical for us, but the result is clear and nice :)

```suggestion
| `cookieOnly` | <pre>boolean</pre> | Instructs Kuzzle to store / use authentication tokens within secured HTTP cookies |
```

Reply:
Also, since you're at it, could you add a documentation for the `pretty` option?

E.g.

```
| `pretty` | <pre>boolean</pre> | Prettify the JSON response, making it more readable for humans |